autoload/fzf/vim.vim

```diff
@@ -163,7 +163,7 @@ function! fzf#vim#with_preview(...)
     call remove(args, 0)
   endif
 
-  if !executable(s:bash())
+  if (has('win32') && &shell !~? '\v<(ba)?sh>') || !executable(s:bash())
```

Comment:
What exactly does this supposed to do?

The original condition checks if a `bash` is available, and if not, does not add preview options.

So you don't want preview window to appear? Is the title of the PR correct?

Reply:
Can you show me some screenshots before/after the patch?